dir2cast.php

```diff
@@ -1246,7 +1246,13 @@ protected function scan()
             if(self::$EMPTY_PODCAST_IS_ERROR && 0 == $item_count)
             {
                 http_response_code(404);
-                throw new Exception("No content yet.");
+                if (0 == $item_count) {
+                    throw new Exception("Item count zero.");
+                }
+                if (self::$EMPTY_PODCAST_IS_ERROR) {
+                    throw new Exception("Self empty podcast error.");
+                }
```

Comment:
because the outer condition is `if(self::$EMPTY_PODCAST_IS_ERROR && 0 == $item_count)`, this will always throw "Item count is zero". The "Self empty podcast error" exception is unreachable.

Reply:
Can't track down the branch, but I had pulled this in from another branch on the main project, just like the attribution changes in the readme. Reverting since I don't see this on the main branch